app/products/playbooks/screens/playbook_run/checklist/checklist_item/checklist_item.tsx

```diff
@@ -172,18 +191,41 @@ const ChecklistItem = ({
         }
     }
 
+    const renderBottomSheet = useCallback(() => (
+        <ChecklistItemBottomSheet
+            item={item}
+            assignee={assignee}
+            onCheck={toggleChecked}
+            onSkip={toggleSkipped}
+            onRunCommand={executeCommand}
+            teammateNameDisplay={teammateNameDisplay}
+        />
+    ), [assignee, executeCommand, item, teammateNameDisplay, toggleChecked, toggleSkipped]);
+
+    const onPress = useCallback(() => {
+        bottomSheet({
+            title: intl.formatMessage({id: 'playbook_run.checklist.taskDetails', defaultMessage: 'Task Details'}),
+            renderContent: renderBottomSheet,
+            snapPoints: [1, '60%', '80%'],
+            theme,
+            closeButtonId: 'close-checklist-item',
+        });
+    }, [intl, renderBottomSheet, theme]);
+
     return (
         <View style={styles.checklistItem}>
             <View style={styles.checkboxContainer}>
                 {checkbox}
             </View>
             <View style={styles.itemDetails}>
-                <View style={styles.itemDetailsTexts}>
-                    <Text style={[styles.itemTitle, skipped && styles.skippedText]}>{item.title}</Text>
-                    {item.description && (
-                        <Text style={[styles.itemDescription, skipped && styles.skippedText]}>{item.description}</Text>
-                    )}
-                </View>
+                <TouchableOpacity onPress={onPress}>
+                    <View style={styles.itemDetailsTexts}>
+                        <Text style={[styles.itemTitle, skipped && styles.skippedText]}>{item.title}</Text>
+                        {item.description && (
```

Comment:
```suggestion
                        {Boolean(item.description) && (
```

This avoids potential errors with rendering